### PR TITLE
fix routing issue for teacher dashboard

### PR DESCRIPTION
**Pull request by jsparvath**
The path is defined in an environment variable that MUST be added in an .env file in the root of the project like so:

`REACT_APP_ROOT_NAME=teacher-react
`

This PR should fix the routing, so the teacher dashboards home exists at "zeeguu/teacher-react", and all subsequent paths follow. 

Also:

- Styling updates
- Uses cookies to store the time period to display data in the users preferred time period

### CODE REVIEW

File: zeeguu-teacher-dashboard/src/App.js [hunk 1]
```diff
@@ -11,13 +12,15 @@ import NotLoggedInPage from './pages/NotLoggedInPage'
 
 import TimePeriodContext from './context/TimePeriodContext'
 import UserContext from './context/UserContext'
+import {useCookie} from './utilities/hooks'
```
Comment (mircealungu) [inline zeeguu-teacher-dashboard/src/App.js]: hooks seems like a strange name for a module from which to import useCookie. or do i mis-understand something?

File: zeeguu-teacher-dashboard/src/assets/styles/components/cohortList.scss [hunk 2]
```diff
@@ -1,8 +1,9 @@
 @import '../shared';
 .cohorts-list {
   display: grid;
-  grid-column-gap: $spacing-unit;
-  grid-row-gap: calc(2 * #{$spacing-unit});
+  grid-gap: calc(4 * #{$spacing-unit});
+  // grid-column-gap: $spacing-unit;
```
Comment (mircealungu) [inline zeeguu-teacher-dashboard/src/assets/styles/components/cohortList.scss]: can we remove the commented lines?

File: zeeguu-teacher-dashboard/src/pages/StudentPage.js [hunk 3]
```diff
@@ -30,51 +30,60 @@ const StudentActivity = ({ studentId }) => {
...
+          <div className="student-activity" key={index}>
+            <p>{day.date}</p>
+            {day.reading_sessions.map((readingSession, index) => (
+              <ExpansionPanel key={index}>
+                <ExpansionPanelSummary expandIcon={<MdExpandMore />}>
+                  <h2 className="student-activity-item-heading">
+                    {readingSession.article_title}
+                  </h2>
+                  <p className="student-activity-item-duration">
+                    {secondsToHoursAndMinutes(readingSession.duration / 1000)}
```
Comment (mircealungu) [inline zeeguu-teacher-dashboard/src/pages/StudentPage.js]: Never have magic constants in code.
Maybe at least move it into a `sessionDurationInSeconds(readingSession)` function?

File: zeeguu-teacher-dashboard/src/App.js (hunk 1 above)
Comment (jsparvath) [reply zeeguu-teacher-dashboard/src/App.js]: Hooks are a React concept to encapsulate reusable logic that is 'hooked' into the React runtime
Its a naming convention to put 'use' in front of the hook

**Pull request merged**